### PR TITLE
Support Vercel

### LINKED ISSUE #174: Deployment to Vercel
Hi, I just tried to push this to Vercel and it looks like it gives an error due to the package.json node engine.

image

Any thoughts on how to get this deployed on Vercel?

It seems to be erroring even before env's are set. 

Thanks!

### PR DESCRIPTION
**Pull request by elliottsj**
Closes #174

### CODE REVIEW

File: docs/Get started.md [hunk 1]
```diff
@@ -177,6 +177,24 @@ Ackee now runs on port `3000` and is only accessible from you local network. It'
 
 Netlify adds a forked version of Ackee to your GitHub repositories. You can always [pull the newest code](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/syncing-a-fork) from the original project. Pushing those changes to your repository will automatically deploy the new version of Ackee on Netlify.
 
+## With Vercel
+
+### 1. Deploy to Vercel
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https%3A%2F%2Fgithub.com%2Felecterious%2FAckee&env=ACKEE_USERNAME,ACKEE_PASSWORD,ACKEE_MONGODB,ACKEE_ALLOW_ORIGIN&envDescription=Environment%20variables%20needed%20to%20configure%20the%20user%20and%20database%20connection&envLink=https%3A%2F%2Fgithub.com%2Felecterious%2FAckee%2Fblob%2Fmaster%2Fdocs%2FOptions.md)
```
Comment (elliottsj) [inline docs/Get started.md:184]: This button was constructed via https://vercel.com/docs/more/deploy-button.

It tells Vercel to prompt the user for required environment variables `ACKEE_USERNAME`, `ACKEE_PASSWORD`, `ACKEE_MONGODB`, and `ACKEE_ALLOW_ORIGIN`.

File: package.json [hunk 2]
```diff
@@ -115,6 +115,6 @@
     }
   },
   "engines": {
-    "node": ">= 14"
+    "node": ">= 12"
```
Comment (elliottsj) [inline package.json:118]: In my brief testing, I didn't notice any crashes or issues on Node.js 12.

File: vercel.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "build": {
+    "env": {
+      "NODEJS_AWS_HANDLER_NAME": "handler"
```
Comment (elliottsj) [inline vercel.json:4]: This is needed so Vercel treats the serverless function as an AWS Lambda function. See https://vercel.com/docs/runtimes#advanced-usage/advanced-node-js-usage/aws-lambda-api and the docstring in `api/index.js`.

File: .gitignore [hunk 4]
```diff
@@ -31,4 +31,7 @@ node_modules
 
 # Dist folder
 dist/*
-!dist/.gitkeep
\ No newline at end of file
+!dist/.gitkeep
+
+# Vercel
+.vercel
```
Comment (elliottsj) [inline .gitignore:37]: When using the [Vercel CLI](https://vercel.com/docs/cli) in this project, a `.vercel` directory is created with some generated files that shouldn't be committed.

**Comment by electerious** [comment]
So good 💯 Big props for the great PR and for adding code comments on GitHub. Will merge it as soon as I have time to test it!

**Review by aleccool213 — APPROVED** [review]

**Pull request merged**